mappings/net/minecraft/entity/ai/brain/ScheduleBuilder.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS avz net/minecraft/entity/ai/brain/ScheduleBuilder
+	CLASS avz$a ActivityData
+		FIELD b activity Lavw;
+		METHOD b getActivity ()Lavw;
+	FIELD b activities Ljava/util/List;
+	METHOD <init> (Lavy;)V
+		ARG 1 schedule
+	METHOD a build ()Lavy;
+	METHOD a addActivity (ILavw;)Lavz;
+		ARG 1 priority
```

Comment:
I may be very wrong, but I thought this is the time of day that the activity starts?

Reply:
Oh, also it may be worth calling this `withActivity` to fit builder pattern.